example/lib/main.dart

```diff
@@ -61,49 +61,52 @@ class MyApp extends StatelessWidget {
             ),
             Spacer(),
             Center(
-              child: FlatButton(
+              child: TextButton(
                 child: Text('Update Context'),
                 onPressed: () {
-                  Posthog.setContext({'custom': 123});
+                  Posthog().setContext({'custom': 123});
                 },
               ),
             ),
             Spacer(),
             Center(
-              child: FlatButton(
+              child: TextButton(
                 child: Text('Clear Context'),
                 onPressed: () {
-                  Posthog.setContext({});
+                  Posthog().setContext({});
                 },
               ),
             ),
             Spacer(),
             Center(
-              child: FlatButton(
+              child: TextButton(
                 child: Text('Disable'),
                 onPressed: () async {
-                  await Posthog.disable();
-                  Posthog.capture(eventName: 'This event will not be logged');
+                  await Posthog().disable();
+                  Posthog().capture(
+                      eventName: 'This event will not be logged',
+                      properties: {});
                 },
               ),
             ),
             Spacer(),
             Center(
-              child: FlatButton(
+              child: TextButton(
                 child: Text('Enable'),
                 onPressed: () async {
-                  await Posthog.enable();
-                  Posthog.capture(eventName: 'Enabled capturing events!');
+                  await Posthog().enable();
+                  Posthog().capture(
```

Comment:
Is this standard Dart indentation? (I don't know so I'm truly asking)

Reply:
Yeah, it's pretty odd for those who are used to other languages, but it's the default, I even used dart format in this file before saving the changes